week-8/Python/Governess-Simpson.py

```diff
@@ -0,0 +1,71 @@
+'''
```

Comment:
This is the code pulled from the HackerEarth challenges, and both passed all three test cases.